docs/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
@chetak123 can you please rename this directory to sig-security-docs to match the previous one under k/community?

It seems a little silly since this is now our sig-security repo, but I'm afraid that a "docs" subdirectory sounds like it's documentation *about* us or something, rather than the name of a subproject like it is.